src/main/kotlin/net/atos/zac/app/admin/model/RestReferenceTableValue.kt

```diff
@@ -13,7 +14,10 @@ import nl.lifely.zac.util.NoArgConstructor
 @NoArgConstructor
 class RestReferenceTableValue(
     var id: Long,
+
+    @field:NotBlank
     var naam: String,
```

Comment:
Would be nice to have a type that doesn't allow for empty string values, then it is clear from the type (name) itself that this field will always contain a value.

Reply:
Yes I agree. However I already spend an awful lot of effort on this quite simple story so maybe for later. :-)